test/test_mbjson.py

```diff
@@ -0,0 +1,182 @@
+            self.json_doc = json.loads(f.read())
+
+    def test_release(self):
+        m = Metadata()
+        a = Album("1")
+        release_to_metadata(self.json_doc, m, a)
+        self.assertEqual(m['album'], 'The Dark Side of the Moon')
+        self.assertEqual(m['albumartist'], 'Pink Floyd')
+        self.assertEqual(m['albumartistsort'], 'Pink Floyd')
+        self.assertEqual(m['asin'], 'b123')
+        self.assertEqual(m['barcode'], '123')
+        self.assertEqual(m['catalognumber'], 'SHVL 804')
+        self.assertEqual(m['date'], '1973-03-24')
+        self.assertEqual(m['label'], 'Harvest')
+        self.assertEqual(m['musicbrainz_albumartistid'], '83d91898-7763-47d7-b03b-b92132375c47')
+        self.assertEqual(m['musicbrainz_albumid'], 'b84ee12a-09ef-421b-82de-0441a926375b')
+        self.assertEqual(m['releasecountry'], 'GB')
+        self.assertEqual(m['releasestatus'], 'official')
+        self.assertEqual(m['script'], 'Latn')
+        self.assertEqual(m['~albumartists'], 'Pink Floyd')
+        self.assertEqual(m['~albumartists_sort'], 'Pink Floyd')
+        self.assertEqual(m['~releaselanguage'], 'eng')
+
+    def test_media_formats_from_node(self):
+        formats = media_formats_from_node(self.json_doc['media'])
+        self.assertEqual(formats, '12" Vinyl')
+
+
+class RecordingTest(unittest.TestCase):
+
+    def setUp(self):
+        config.setting = settings
+        self.json_doc = None
+        with open(os.path.join('test', 'data', 'ws_data', 'recording.json')) as f:
+            self.json_doc = json.loads(f.read())
+
+    def test_recording(self):
+        m = Metadata()
+        t = Track("1")
+        recording_to_metadata(self.json_doc, m, t)
+        self.assertEqual(m['artist'], 'Ed Sheeran')
+        self.assertEqual(m['artists'], 'Ed Sheeran')
+        self.assertEqual(m['artistsort'], 'Sheeran, Ed')
+        self.assertEqual(m['isrc'], 'GBAHS1400099')
+        self.assertEqual(m['language'], 'eng')
+        self.assertEqual(m['musicbrainz_artistid'], 'b8a7c51f-362c-4dcb-a259-bc6e0095f0a6')
+        self.assertEqual(m['musicbrainz_recordingid'], 'cb2cc207-8125-445c-9ef9-6ea44eee959a')
+        self.assertEqual(m['musicbrainz_workid'], 'dc469dc8-198e-42e5-b5a7-6be2f0a95ac0')
+        self.assertEqual(m['performer:'], 'Ed Sheeran')
+        self.assertEqual(m['performer:vocals'], 'Ed Sheeran')
+        self.assertEqual(m['title'], 'Thinking Out Loud')
+        self.assertEqual(m['work'], 'Thinking Out Loud')
+        self.assertEqual(m['writer'], 'Ed Sheeran; Amy Wadge')
+        self.assertEqual(m['~artists_sort'], 'Sheeran, Ed')
+        self.assertEqual(m['~length'], '4:41')
+        self.assertEqual(m['~recordingtitle'], 'Thinking Out Loud')
+
+
+class TrackTest(unittest.TestCase):
+
+    def setUp(self):
+        config.setting = settings
+        self.json_doc = None
+        with open(os.path.join('test', 'data', 'ws_data', 'track.json')) as f:
+            self.json_doc = json.loads(f.read())
```

Comment:
`json.load(f)` again
`os.path.join('test', 'data', 'ws_data')` is repeated at lot

I think you could do something like (untested):

```python
class JsonTests(object):
    def setup(self, filename):
        config.setting = settings
        self.json_doc = None
        with open(os.path.join('test', 'data', 'ws_data', filename)) as f:
            self.json_doc = json.load(f)

class TrackTest(unittest.TestCase, JsonTests):

    def setUp(self):
        self.setup('track.json')
...
````